extensions/amp-analytics/0.1/variables.js

```diff
@@ -211,6 +211,25 @@ export class VariableService {
     this.register_('LINKER_PARAM', (name, id) =>
       this.linkerReader_.get(name, id)
     );
+
+    // Was set async before
+    this.register_('FIRST_CONTENTFUL_PAINT', () => {
```

Comment:
Are there tests on these three macros? Do they still apply?

Reply:
Oops, forgot to add tests.